Problem: unique-binary-search-trees-ii
Submission ID: 1695676054
Submission Time: 2025-07-12 14:28:33
============================================================

State: SUCCESS
Status: Runtime Error
Language: Python

Runtime: N/A ms
Memory: 12.49 MB

Test Cases: 0/8 passed

Last Test Case:
Input: 3
Expected: [[1,null,2,null,3],[1,null,3,2],[2,1,3],[3,1,null,null,2],[3,2,null,1]]

============================================================
RAW SUBMISSION DATA
============================================================
{
  "status_code": 15,
  "lang": "python",
  "run_success": false,
  "runtime_error": "Line 86: TypeError: [[1, '#', 2, '#', 3], [1, '#', 3, 2], [2, 1, 3], [3, 1, '#', '#', 2], [3, 2, '#', 1]] is not valid value for the expected return type list<TreeNode>",
  "full_runtime_error": "TypeError: [[1, '#', 2, '#', 3], [1, '#', 3, 2], [2, 1, 3], [3, 1, '#', '#', 2], [3, 2, '#', 1]] is not valid value for the expected return type list<TreeNode>\n    raise TypeError(str(ret) + \" is not valid value for the expected return type list<TreeNode>\");\nLine 86 in _driver (Solution.py)\n    _driver()\nLine 94 in <module> (Solution.py)",
  "status_runtime": "N/A",
  "memory": 12488000,
  "question_id": "95",
  "elapsed_time": 66,
  "compare_result": "00000000",
  "code_output": "",
  "std_output": "",
  "last_testcase": "3",
  "expected_output": "[[1,null,2,null,3],[1,null,3,2],[2,1,3],[3,1,null,null,2],[3,2,null,1]]",
  "task_finish_time": 1752355711407,
  "task_name": "judger.judgetask.Judge",
  "finished": true,
  "total_correct": 0,
  "total_testcases": 8,
  "runtime_percentile": null,
  "status_memory": "N/A",
  "memory_percentile": null,
  "pretty_lang": "Python",
  "submission_id": "1695676054",
  "status_msg": "Runtime Error",
  "state": "SUCCESS"
}